src/theory/quantifiers/expr_miner.cpp

```diff
@@ -52,18 +52,24 @@ Node ExprMiner::convertToSkolem(Node n)
 
 void ExprMiner::initializeChecker(std::unique_ptr<SolverEngine>& checker,
                                   Node query)
+{
+  initializeChecker(checker, query, options(), logicInfo());
+}
+
+void ExprMiner::initializeChecker(std::unique_ptr<SolverEngine>& checker,
+                                  Node query,
+                                  const Options& opts,
+                                  const LogicInfo& logicInfo)
 {
   Assert (!query.isNull());
   if (Options::current().quantifiers.sygusExprMinerCheckTimeoutWasSetByUser)
```

Comment:
Can we use the `opts` object here instead of using `options::` and `Options::current()`.

Reply:
Yeah, I'll clean this on a separate PR, this is old.